### PR TITLE
test: improve test coverage for compare

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

**Comment by IyadhKhalfallah** [comment]
> ## [Codecov](https://codecov.io/gh/jina-ai/now/pull/1208?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai) Report
> > Merging [#1208](https://codecov.io/gh/jina-ai/now/pull/1208?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai) ([e2b1275](https://github.com/jina-ai/now/commit/e2b1275767f6ae5a9e062fa2837f631d9d54305e)) into [main](https://codecov.io/gh/jina-ai/now/commit/2730af6a3d11132c8a76f436af3a469d1ce93c3d?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai) ([2730af6](https://github.com/jina-ai/now/commit/2730af6a3d11132c8a76f436af3a469d1ce93c3d)) will **decrease** coverage by `0.12%`.
> > The diff coverage is `75.00%`.
> 
> ```diff
> @@            Coverage Diff             @@
> ##             main    #1208      +/-   ##
> ==========================================
> - Coverage   52.42%   52.31%   -0.12%     
> ==========================================
>   Files          85       85              
>   Lines        4660     4645      -15     
> ==========================================
> - Hits         2443     2430      -13     
> + Misses       2217     2215       -2     
> ```
> 
> Flag	Coverage Δ	
> now	`52.31% <75.00%> (-0.12%)`	⬇️
> Flags with carried forward coverage won't be shown. [Click here](https://docs.codecov.io/docs/carryforward-flags?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai#carryforward-flags-in-the-pull-request-comment) to find out more.
> 
> [Impacted Files](https://codecov.io/gh/jina-ai/now/pull/1208?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai)	Coverage Δ	
> [now/run_all_k8s.py](https://codecov.io/gh/jina-ai/now/pull/1208?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai#diff-bm93L3J1bl9hbGxfazhzLnB5)	`62.92% <74.07%> (+26.92%)`	⬆️
> [now/cli/__init__.py](https://codecov.io/gh/jina-ai/now/pull/1208?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai#diff-bm93L2NsaS9fX2luaXRfXy5weQ==)	`60.71% <100.00%> (+4.04%)`	⬆️
> ... and [15 files with indirect coverage changes](https://codecov.io/gh/jina-ai/now/pull/1208/indirect-changes?src=pr&el=tree-more&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai)
> 
> Help us with your feedback. Take ten seconds to tell us [how you rate us](https://about.codecov.io/nps?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai). Have a feature suggestion? [Share it here.](https://app.codecov.io/gh/feedback/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jina-ai)

Not sure why the coverage is down, which shouldn't be. I'm investigating this.

File: now/run_all_k8s.py [hunk 1]
```diff
@@ -62,7 +63,10 @@ def get_docarray(dataset):
         print(f'Loading queries from {dataset}')
         return DocumentArray.load_binary(dataset)
     print(f'Pulling queries from {dataset}')
-    return DocumentArray.pull(name=dataset, show_progress=True)
+    da = DocumentArray.pull(name=dataset, show_progress=True)
+    if 'NOW_CI_RUN' in os.environ:
+        da = da[:5]
```
Comment (joschkabraun) [inline now/run_all_k8s.py:68]: if you change this line to `da = da[:21]` we get also these lines: https://app.codecov.io/gh/jina-ai/now/pull/1208/blob/now/compare/compare_flows.py#L66

**Review by joschkabraun — CHANGES_REQUESTED** [review]
Small suggestion, otherwise LGTM!

**Review by Tanguyabel — APPROVED** [review]
one little comment about the codecov version, otherwise looks goot to me!

**Review by joschkabraun — APPROVED** [review]
Nice work!

**Pull request merged**